packages/core/data-transfer/src/strapi/providers/remote-destination/index.ts

```diff
@@ -172,29 +197,31 @@ class RemoteStrapiDestinationProvider implements IDestinationProvider {
     return new Writable({
       objectMode: true,
       final: async (callback) => {
-        const e = await this.#dispatchTransfer('assets', null);
+        // TODO: replace this stream call by an end call
```

Comment:
Is this important before merging or something that can wait?

Reply:
It can wait, it'll be for when we'll handle locking/unlocking the transfer ops in the remote handler